cookbook/case_studies/ml_training/house_price_prediction/multiregion_house_price_predictor.py

```diff
@@ -1,19 +1,29 @@
 """
-Predicting House Price in Multiple Regions with XGBoost and Dynamic Workflows
-------------------------------------------------------------------------------
+Predicting House Price in Multiple Regions Using XGBoost and Dynamic Workflows
+-------------------------------------------------------------------------------
+In this tutorial, we will understand how to predict house prices in multiple regions using XGBoost, and :ref:`dynamic workflows <sphx_glr_auto_core_control_flow_dynamics.py>` in Flyte.
 
-In this example, you'll use the house price prediction model for one region to expand it to multiple regions. 
+We will split the generated dataset into train, test and validation set. 
+
+Next, we will create two dynamic workflows in Flyte, that will:
+
+1. Generate and split the data for multiple regions.
+
+2. Train the model using XGBoost and generate predictions.
+
+We can use two different methods to fit the model and generate predictions, but including them in the same dynamic workflow will parallelize the tasks together, where every branch denotes a region.
```

Comment:
Can you please move this sentence to the required function wherever this happens? I have mentioned this before as well; you may have missed it.